src/openApiClientJs/README.md

```diff
@@ -0,0 +1,17 @@
+# Generate TypeScript client for Joanie api
+
```

Comment:
Can you just describe the purpose of this package.

Furthermore, can you into `docs/explanation` a new markdown file addressed to frontend developer to explain that Joanie offers the feature to generate TS types for its api.